src/openrct2/util/SawyerCoding.cpp

```diff
@@ -109,14 +104,14 @@ size_t sawyercoding_decode_sc4(const uint8_t* src, uint8_t* dst, size_t length,
     size_t decodedLength = decode_chunk_rle_with_size(src, dst, length - 4, bufferLength);
 
     // Decode
-    for (size_t i = 0x60018; i <= std::min(decodedLength - 1, static_cast<size_t>(0x1F8353)); i++)
+    for (size_t i = 0x60018; i <= std::min(decodedLength - 1, (size_t)0x1F8353); i++)
```

Comment:
dont convert a static_cast into a C-Style cast

Reply:
Don't convert any named cast into a C-Style cast (this repeats throughout the change).

Named casts are safer than C-Style casts.

Reply:
Something screwed up when I checkout the code, I've never convert these back to C-style...